### PR TITLE
Added italian language

### PR DESCRIPTION
**Pull request by filipporomani**

### CODE REVIEW

**Review by staniel359 — APPROVED** [review]
@filipporomani Great job, thank you.

**Comment by filipporomani** [comment]
Done! Now all is ok!

**Review by staniel359 — APPROVED** [review]
@filipporomani Nice, thank you.

File: src/plugins/i18n.js [hunk 1]
```diff
@@ -3,10 +3,12 @@ import {
 } from 'vue-i18n'
 import en from '*/locales/en.json'
 import ru from '*/locales/ru.json'
+import it from '*/locales/it.json'
```
Comment (staniel359) [inline src/plugins/i18n.js:5]: @filipporomani Actually could you please place `it` after `en`?
Languages should go alphabetically.

**Comment by filipporomani** [comment]
No prob! Glad to help! Do you have a discord server?

**Review by staniel359 — CHANGES_REQUESTED** [review]

File: src/plugins/i18n.js (hunk 1 above)
Comment (filipporomani) [reply src/plugins/i18n.js:5]: Sure

Comment (filipporomani) [reply src/plugins/i18n.js:5]: Pusehd again! fixed

**Review by staniel359 — APPROVED** [review]

**Comment by staniel359** [comment]
@filipporomani 
>Do you have a discord server?

No, I don't. Why?

**Comment by filipporomani** [comment]
> @filipporomani
> 
> > Do you have a discord server?
> 
> No, I don't. Why?

Just to ask, I was curious to join xD!

**Comment by staniel359** [comment]
@filipporomani I see.
Maybe in future.
Thank you for your help.

**Comment by filipporomani** [comment]
No prob! Contact me by email, mail@filipporomani.it if you need any further help with development/translations!

**Comment by staniel359** [comment]
@filipporomani Thank you.

**Pull request merged**